### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by kreativedev**
Fixed typos on Subtraction and Multiplication

### CODE REVIEW

File: run2.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
+import argparse
```
Comment (januschung) [inline run2.py:1]: Is this file being added by mistake?

**Review by januschung — CHANGES_REQUESTED** [review]
Other than the file run2.py, lgtm

File: run2.py (hunk 1 above)
Comment (kreativedev) [reply run2.py:1]: Was a total mistake, didn't even notice i accidentally committed that one

Comment (kreativedev) [reply run2.py]: removed the accidentally added file

**Pull request closed without merging**